release_bot/releasebot.py

```diff
@@ -104,40 +105,44 @@ def find_open_release_issues(self):
         Looks for opened release issues on github
         :return: True on found, False if not found
         """
-        cursor = ''
         release_issues = {}
         latest_version = Version(self.github.latest_release())
-        while True:
-            edges = self.github.walk_through_open_issues(start=cursor, direction='before')
-            if not edges:
-                self.logger.debug(f'No more open issues found')
-                break
-            else:
-                for edge in reversed(edges):
-                    cursor = edge['cursor']
-                    title = edge['node']['title'].lower().strip()
-                    match, version = process_version_from_title(title, latest_version)
-                    if match:
-                        if edge['node']['authorAssociation'] in ['MEMBER', 'OWNER',
-                                                                 'COLLABORATOR']:
-                            release_issues[version] = edge['node']
-                            self.logger.info(f'Found new release issue with version: {version}')
+        opened_issues = self.project.get_issue_list(IssueStatus.open)
+        if not opened_issues:
+            self.logger.debug(f'No more open issues found')
+        else:
+            for issue in opened_issues:
+                match, version = process_version_from_title(issue.title, latest_version)
+                if match:
+                    if (self.project.can_close_issue(self.conf.github_username, issue)
+                            or self.project.can_close_issue(self.conf.pagure_username, issue)):
+                        release_issues[version] = issue
+                        self.logger.info(f'Found new release issue with version: {version}')
+                    else:
+                        if self.which_service() == "Github":
+                            self.logger.warning(f"User {self.conf.github_username } "
+                                                f"has no permission to modify issue")
                         else:
-                            self.logger.warning(
-                                f"Author association {edge['node']['authorAssociation']!r} "
-                                f"not in ['MEMBER', 'OWNER', 'COLLABORATOR']")
+                            self.logger.warning(f"User {self.conf.pagure_username} "
+                                                f"has no permission to modify issue")
```

Comment:
I like the `which_service()` method. What about to have one for getting username, like:
```python
def which_username():
  if self.which_service() == "Github":
    return self.conf.github_username
  else if self.which_service() == "Pagure":
    return self.conf.pagure_username
  return None
```
then you could replace
```python
if (self.project.can_close_issue(self.conf.github_username, issue)
   or self.project.can_close_issue(self.conf.pagure_username, issue)):
```
with
```python
if (self.project.can_close_issue(self.which_username(), issue):
```
and replace
```python
                        if self.which_service() == "Github":
                            self.logger.warning(f"User {self.conf.github_username } "
                                                f"has no permission to modify issue")
                        else:
                            self.logger.warning(f"User {self.conf.pagure_username} "
                                                f"has no permission to modify issue")
```
with
```python
                            self.logger.warning(f"User {self.which_username()} "
                                                f"has no permission to modify issue")
```